### PR TITLE
HV: patch set to enable hybrid scenario

### PR DESCRIPTION
**Pull request by jsun26intel**
Jason Chen CJ (1):
  HV: change nuc7i7bnh ram start to 0x60000000

Victor Sun (7):
  HV: fix wrong log when vlapic process init sipi
  HV: return -1 with invalid vcpuid in pt icr access
  HV: remove cpu_num from vm configurations
  HV: remove unused INVALID_VM_ID
  HV: correct target vm for hcall gpa to hpa 
  HV: add hybrid scenario
  HV: return virtual lapic id in vcpuid 0b leaf

Tracked-On: #3214

Signed-off-by: Victor Sun <victor.sun@intel.com>
Reviewed-by: Jason Chen CJ <jason.cj.chen@intel.com>

### CODE REVIEW

**Comment by acrnsi** [comment]
OK to verify

**Comment by HuiHuangShi** [comment]
out of time

**Comment by acrnsi** [comment]
OK to verify

**Comment by HuiHuangShi** [comment]
2 new potential violations to the coding guideline detected. Please help check. Thanks!<br><1> config:sharing_config filename:/hypervisor/arch/x86/guest/vm.c func:prepare_vm offset:708<br> reason:Pointer not checked for null before use. : vm<br><2> config:part_config filename:/hypervisor/arch/x86/guest/vm.c func:prepare_vm offset:708<br> reason:Pointer not checked for null before use. : vm<br><br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by HuiHuangShi** [comment]
out of time

**Comment by wenlingz** [comment]
@dongyaozu please help approve this PR which is for V1.1, thanks

File: hypervisor/common/hypercall.c [hunk 1]
```diff
@@ -805,7 +805,14 @@ int32_t hcall_gpa_to_hpa(struct acrn_vm *vm, uint16_t vmid, uint64_t param)
 {
 	int32_t ret = -1;
 	struct vm_gpa2hpa v_gpa2hpa;
-	struct acrn_vm *target_vm = get_vm_from_vmid(vmid);
```
Comment (dongyaozu) [inline hypervisor/common/hypercall.c]: This solution is not good enough. get_vm_from_vmid should work fine, but the vm_id of SOS changes. This is same for other VMs too.

**Comment by HuiHuangShi** [comment]
2 new potential violations to the coding guideline detected. Please help check. Thanks!<br><1> config:sharing_config filename:/hypervisor/arch/x86/guest/vm.c func:prepare_vm offset:708<br> reason:Pointer not checked for null before use. : vm<br><2> config:part_config filename:/hypervisor/arch/x86/guest/vm.c func:prepare_vm offset:708<br> reason:Pointer not checked for null before use. : vm<br><br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Pull request merged**